app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,8 @@
+class TagsController < ApplicationController
+
+  def show
+    @tag = Tag.find_by(name: params[:id])
+    render json: @tag, include: "notes.tags"
```

Comment:
The `include: "notes.tags"` is needed here to force our serializer to go one level deep - so the tags are included in the notes display.